src/features/channels/components/channel-list/about/about.tsx

```diff
@@ -1,20 +1,42 @@
+import useOnMount from '@metis/common/hooks/useOnMount';
+import httpService from '@metis/common/services/http.service';
+import Grid from '@mui/material/Grid';
 import Dialog from '@mui/material/Dialog';
 import DialogContent from '@mui/material/DialogContent';
 import DialogContentText from '@mui/material/DialogContentText';
 import DialogTitle from '@mui/material/DialogTitle';
+import Divider from '@mui/material/Divider';
 import Typography from '@mui/material/Typography';
-import * as React from 'react';
+import { useState, useEffect } from 'react';
 import useStyles from './about.styles';
 
+export interface WebAppInfo {
+  name: string;
+  version: string;
+}
+
 type Props = {
   title: string;
   message: string;
   onClick?: () => void;
 };
 
+const loadWebAppInfo = async (): Promise<WebAppInfo[]> => {
+  const response = await httpService.get<WebAppInfo[]>('/v1/api/version');
+  return response.data;
```

Comment:
Just a suggestion, you can destructure the response object and return the data directly. Ex: 
```suggestion
const { data } = await httpService.get<WebAppInfo[]>('/v1/api/version');
return data;
```

Reply:
I will consider it